File: hw/bohorquez/U3/HW35SingletonTax/src/main/java/ec/edu/espe/hw34calculatorustax/view/Calculator.java
package ec.edu.espe.hw34calculatorustax.view;

import ec.edu.espe.hw34calculatorustax.model.USTax;

/**
 *
 * @author Brandon Pazmino
 */
public class Calculator {
    public static void main(String[] args) {
        USTax tax = USTax.getInstance();
        float subtotal = 10.0F;
        float total = tax.salesTotal(subtotal);
        System.out.println("The total is --> "+ total);
        
        USTax newTaxInstance = new USTax(0.14F);
        total = newTaxInstance.salesTotal(subtotal);
        System.out.println("The total IVA is --> "+ total + "%");
    }
        
}

File: hw/pazmino/u3/HW35SingletonUSTax/hw34CalculatorUSTaxPython/view/calculator.py
from model.us_tax import USTax

def main():
    tax = USTax.get_instance()
    subtotal = 10.0
    total = tax.sales_total(subtotal)
    print(f"The total is --> {total}")

    new_tax_instance = USTax(0.14)
    total = new_tax_instance.sales_total(subtotal)
    print(f"The total IVA is --> {total}%")

if __name__ == "__main__":
    main()
